src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -154,6 +157,24 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
     }
   }
 
+  private void configureProxy(ElasticsearchSinkConnectorConfig config,
+                              HttpClientConfig.Builder builder) {
+    final String proxy =
+            config.getString(ElasticsearchSinkConnectorConfig.CONNECTION_PROXY_CONFIG);
+    if (proxy != null && !proxy.isEmpty()) {
+      try {
+        String protocol = proxy.split("://")[0];
```

Comment:
Let's use the `URL.getProtocol`, `.getHost`, `getPort`. https://docs.oracle.com/javase/7/docs/api/java/net/URL.html